app/package.json

```diff
@@ -25,6 +25,14 @@
     "Desktop app",
     "InstantMessaging"
   ],
+  "build": {
```

Comment:
This should go into the development package.json i.e. - `zulip-electron/package.json`. Read more about the structure [here](https://www.electron.build/tutorials/two-package-structure).